### PR TITLE
maintenance-sprint-onvirtual-cards

### PR DESCRIPTION
**Pull request by jasnvella2**
This PR Addresses: https://weavr-payments.atlassian.net/browse/DEV-6430

### CODE REVIEW

**Review by oleggrech-t — APPROVED** [review]

**Review by PeterThrels — APPROVED** [review]

**Comment by gaucj-t** [comment]
@jasnvella2 can we upgrade to node 20 also ?

**Review by gaucj-t — APPROVED** [review]

File: components/registration/RegisterForm.vue [hunk 1]
```diff
@@ -167,7 +167,7 @@ export default class RegisterForm extends mixins(BaseMixin, ValidationMixin) {
         try {
             this.$v.$touch()
             if (this.$v.$invalid || !this.isPasswordValid) {
-                return
+                // empty
```
Comment (gaucj-t) [inline components/registration/RegisterForm.vue:170]: is this correct ?

**Review by gaucj-t — APPROVED** [review]

File: components/registration/RegisterForm.vue (hunk 1 above)
Comment (jasnvella2) [reply components/registration/RegisterForm.vue:170]: previously build was failing due to having an empty return

`// empty `as suggested by eslint, seemed to solve it

Comment (gaucj-t) [reply components/registration/RegisterForm.vue:170]: You would still need an return to break out of the conditions

Comment (jasnvella2) [reply components/registration/RegisterForm.vue]: fixed

**Pull request closed without merging**